import-scripts/oncokb-annotator.sh

```diff
@@ -143,13 +145,13 @@ fi
 
 # Only commit if all steps succeeded
 if [ $ONCOKB_ANNOTATION_SUCCESS -eq 0 ] ; then
-    cd $MSK_SOLID_HEME_DATA_HOME ; $HG_BINARY update -C ; find . -name "*.orig" -delete
```

Comment:
only one set of changes/updates happening in oncokb annotator script so running this is okay.